modular_chomp/code/modules/mob/living/simple_mob/subtypes/vore/gryphon.dm

```diff
@@ -0,0 +1,266 @@
+
+	friendly = list("beak-nudges", "purrs towards", "slobberlicks")
+	vore_bump_emote = "tries to quickly devour"
+
+	old_x = -16
+	old_y = 0
+	default_pixel_x = -16
+	pixel_x = -16
+	pixel_y = 0
+	mount_offset_y = 15
+	mount_offset_x = -12
+	max_buckled_mobs = 1
+	can_buckle = TRUE
+	buckle_movable = TRUE
+	buckle_lying = FALSE
+
+	var/random_skin = TRUE
+	var/list/skins = list(
+		"gryphon"
+	)
+
+	special_attack_min_range = 2
+	special_attack_max_range = 4
+	special_attack_cooldown = 30 SECONDS
+
+	devourable = FALSE
+
+	ai_holder_type = /datum/ai_holder/simple_mob/vore/gryphon
+	say_list_type = /datum/say_list/gryphon
+
+	can_be_drop_prey = FALSE
+	var/leap_warmup = 2 SECOND // How long the leap telegraphing is.
+	var/leap_sound = 'sound/weapons/spiderlunge.ogg'
+
+// Activate Noms!
+/mob/living/simple_mob/vore/gryphon
+	vore_bump_chance = 25
+	vore_digest_chance = 100
+	// Strong pounce, but needs to be alone to trigger it
+	vore_pounce_chance = 100
+	vore_pounce_falloff = 0
+	vore_active = TRUE
+	vore_icons = 0
+	vore_capacity = 2
+	swallowTime = 20
+	vore_ignores_undigestable = TRUE
+	vore_default_mode = DM_DIGEST
+	vore_pounce_maxhealth = 125
+	vore_bump_emote = "tries to gulp down"
+	vore_capacity_ex = list("stomach" = 1, "throat" = 1)
+	vore_fullness_ex = list("stomach" = 0, "throat" = 0)
+	vore_icon_bellies = list("stomach", "throat")
+
+/datum/ai_holder/simple_mob/vore/gryphon
+	var/eat_attempts = 0
+	var/maybe_eating = null
+
+/datum/ai_holder/simple_mob/vore/gryphon/find_target(list/possible_targets, has_targets_list)
+	ai_log("find_target() : Entered.", AI_LOG_TRACE)
+	var/targets = list()
+	if(!has_targets_list)
+		possible_targets = list_targets()
+	var/mob/sentient = null
+	var/alone = TRUE
+	for(var/mob/possible_target in possible_targets)
+		if(can_attack(possible_target)) // Can we attack it?
+			targets += possible_target
+			if (possible_target.client != null)
+				if (sentient == null)
+					if (vore_check(possible_target))
+						sentient = possible_target
+					else
+						alone = FALSE
+				else
+					alone = FALSE
+
+	// Only one player, consider eating them
+	if (sentient != null)
+		return possibly_eat(sentient, alone)
+
+	// Only one mob, eat them
+	if (length(targets) == 1 && alone)
+		var/target = pick_target(targets)
+		if (vore_check(target) && can_attack(target))
+			give_target(target)
+			return target
+	eat_attempts = 0
+	return null
+
+/datum/ai_holder/simple_mob/vore/gryphon/proc/possibly_eat(var/target, var/alone)
+	if (target != maybe_eating)
+		eat_attempts = 0
+
+	maybe_eating = target
+	if (eat_attempts == 5)
+		to_chat(target, span_danger("\The [holder] licks its beak"))
+	else if (eat_attempts == 10)
+		to_chat(target, span_danger("\The [holder]'s stomach grumbles loudly"))
+	else if (alone && eat_attempts > 15 && can_attack(target))
+		give_target(target)
+
+	if (eat_attempts < 11 || alone)
+		eat_attempts += 1
+
+/mob/living/simple_mob/vore/gryphon/init_vore()
+	if(!voremob_loaded)
```

Comment:
Is this var ever even set to TRUE?

Reply:
Do you mean voremob_loaded?

Reply:
yes, spawn one in and take a look, usually should after initializing

Reply:
It definitely is because the gryphon has the proper bellies and works fine

Reply:
Give it a VR test, I highly feel like those were never re-enabled and that had already been fixed.

Reply:
Confirmed, it was fixed

Reply:
Likely can enable the dragon too then after a quick test